bench/macro/lsm-tree-bench-wp8.hs

```diff
@@ -548,6 +551,11 @@ pipelinedIterations :: (Int -> LookupResults -> IO ())
                     -> LSM.TableHandle IO K V B
                     -> IO ()
 pipelinedIterations output !initialSize !batchSize !batchCount !seed tbl_0 = do
+    n <- getNumCapabilities
+    unless (n == 2) $
+      printf "WARNING: the pipelined benchmark is running with %d capabilities \
+             \but it is recommended to run with 2. Pass the -N2 RTS option.\n" n
```

Comment:
Well, yes and no. It's also interesting to see what performance it has if one runs both threads on one core. There's still concurrency available there.

Reply:
I've changed to an info message that just tells the user how many threads are in use